common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -1130,4 +1130,24 @@ lights_vengeance = {
 	artifacts = {
 		lights_vengeance = { value = 1 }
 	}
+}
+justice_gaze = {
+	max_amount = 1
+	spawn_date = 580.1.1
+
+	spawn_chance = {
+		value = 100
+	}
+
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60017						# Alexandros Mograine
+		}
+
```

Comment:
To make sure...
```suggestion
		additive_modifier = {
			value = 100
			character = 60017						# Alexandros Mograine
		}
		additive_modifier = {
			value = 100
			NOT = { c_60017 = { is_alive = yes } }
			
			has_landed_title = d_order_of_the_silver_hand
		}
```

Reply:
so, if alexandros is dead at game start, it shall be given to the silver hand leader? In case of later start dates?

Reply:
Yes.

Reply:
done